changelog.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## What's New (02/18/2020)
+
+### New validation rules
+- OperationIdRequired rule - each operation must has a unique operationId.
+
+### Changed Rule
+- Changed the linter rule BodyTopLevelProperties -- support `systemData` top level property
+- Changed the linter rule PatchBodyParametersSchema -- allow required property which be marked as descrimintor 
```

Comment:
Could you work with our PM with the change log? I saw a couple of grammar issue in the above sentence.